src/amqp-client.cr

```diff
@@ -113,17 +117,17 @@ class AMQP::Client
   def connect : Connection
     if @host.starts_with? '/'
       socket = connect_unix
-      Connection.start(socket, @user, @password, @vhost, @channel_max, @frame_max, @heartbeat, @name)
+      Connection.start(socket, @user, @password, @vhost, @channel_max, @frame_max, @heartbeat, @connection_information, @name)
```

Comment:
looks like `@name` and `@connection_information` are swapped here?

Reply:
No, should be alright. `@connection_information` comes before `@name` because name has a default value in `connection.cr` `self.start`, then they are sent in in the other order to the private method, I now switched them there as well to avoid confusion